client/payment-details/timeline/map-events.js

```diff
@@ -223,6 +223,48 @@ const composeFXString = ( event ) => {
 	);
 };
 
+/**
+ * Returns an array containing fee breakup.
+ *
+ * @param {Object} event Event object
+ *
+ * @return {Array} Array of formatted fee strings
+ */
+const feeBreakup = ( event ) => {
+	if ( ! event?.fee_rates?.history ) {
+		return [];
+	}
+
+	const {
+		fee_rates: { history },
+	} = event;
+
+	const feeLabelMapping = {
+		base: __( '\u00A0 • Base fee %s%%', 'woocommerce-payments' ),
```

Comment:
Is there a reason you choose to add empty spaces and bullets manually here? I don't see any reason not to use an actual unordered list (`<ul>`) with list items here.

I tested it and saw an empty line, wondering what it is, turns out that it's an unsupported discount fee, as @htdat mentioned in https://github.com/Automattic/woocommerce-payments/pull/2500#pullrequestreview-714501248

<img width="406" alt="image" src="https://user-images.githubusercontent.com/5311119/127297141-005f614b-90f1-4d95-85b9-f7ed751a2809.png">

Reply:
I was following the string-based approach, but I like your suggestion better, fixed in: https://github.com/Automattic/woocommerce-payments/pull/2500/commits/25fc362ff7b03b354fab7b7117e6aeb4ef15ccce